go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/RedHatInsights/chrome-service-backend
 
-go 1.20
+go 1.21
```

Comment:
Just make sure we have a build image for this version. Not sure if 21 is available yet.

Reply:
I am testing things out, because I used the `slices` package for some things, but the tests are failing and I assume it's because of this. I'll try to revert every dependency upgrade back to what we had before and see if that solves the issues, and then figure out the `slices` and the `21` version thing.

Reply:
The image of the Dockerfile gets built no problem, so we're safe here! Sources uses `1.23` with `ubi9` as well, so I'd say we're safe here.